.github/workflows/docs.py

```diff
@@ -0,0 +1,36 @@
+"""Script to generate docs based on given input directory (first argument)
+and output directory (second argument)"""
+import os
+import subprocess
+import sys
+import glob
+import nbformat
+
+code_path = sys.argv[1]
+out_path = sys.argv[2]
+
+os.chdir(code_path)
+os.environ["PDOC_ALLOW_EXEC"] = "1"
+
+for notebook_path in glob.glob("examples/PySDM_examples/*/*.ipynb"):
```

Comment:
we could use the `devops_tests` `find_files` function that limits the search to files tracked in git: https://github.com/open-atmos/devops_tests/blob/8eb4d96a1ea8afa09a409a99dccbd0ddd48f80a7/utils.py#L10

Reply:
let's leave this for later